pep-0001.txt

```diff
@@ -94,11 +94,12 @@ the currently active Python core team members described in PEP 13 [5]_.
 Python's BDFL
 -------------
 
-This PEP still uses the title "BDFL-Delegate" for PEP decision makers. This is
+This PEP earlier used the title "BDFL-Delegate" for PEP decision makers. This was
```

Comment:
How about: "Previous versions of this PEP used the title..." ?

Reply:
```suggestion
Previous versions of this PEP used the title "BDFL-Delegate" for PEP decision makers. This was
```